### ISSUE TITLE
Mosses (class Bryopsida in Bryophyta) classification in COL seriously out of date

### ISSUE THREAD

**Issue by TonyRees** [CatalogueOfLife/data#416]
Hi all, I was checking some moss names (phylum: [Bryophyta](https://www.catalogueoflife.org/data/taxon/BRY), class: [Bryopsida](https://www.catalogueoflife.org/data/taxon/8K) and discovered that these are all using a version of MOST (Mosses Moss TROPICOS Database, version 1 Jul 2004) that is seriously out of date. For example the family Seligeriaceae is listed in order "Seligerales" - a misspelling for Seligeriales, which anyway is no longer recognised (in Grimmiales in current MOST/Tropicos i.e. 2022 version).

Is there anything in train to get a more updated classification from MOST/Tropicos maybe...

(Also the version presently in COL has no extinct moss taxa - not sure if those are now in Tropicos or not).

Regards - Tony Rees

**Comment by yroskov** [comment]
I am trying to get MOST update since 2007. Dataset of 2014 had serious problems with classification integrity, and it was not suitable for use in CoL. 

I hope, we'll be able to get global checklist via WFO, when it will be ready.

**Comment by mdoering** [comment]
I wrote to John Brinda with whom I was in touch some time ago. Let's see what he thinks.

**Comment by mdoering** [comment]
John is developing a new version of the list and already provided early versions of it to WFO and the [Consortium of North American Bryophyte Herbaria](https://bryophyteportal.org/)

Website for exploring the data: https://www.bryonames.org

> There is also a basic query interface for accessing the data programmatically.
> For now it is only complete to the species level, but I hope to have the infraspecifics done in a few months time.
> 
> It would be great to see the CoL/GBIF data updated. A lot has changed since 2004...
>

**Comment by yroskov** [comment]
@olafbanki, CoL may try to get data directly from The Bryophyte Nomenclator, or try to get them via WFO. Please, let me know, what is a preferred way.

**Comment by sergbolshakov** [comment]
Hello, is there any news on the update of mosses classification in COL? Meanwhile, Bryonames continues to make good progress.

**Comment by yroskov** [comment]
Still waiting for a new version of MOST. Any updates on it, @mdoering?

**Comment by mdoering** [comment]
Yes. It is here now: https://www.checklistbank.org/dataset/170394/about

**Comment by yroskov** [comment]
Good! We'll update it in December edition.
